lib/bindings/langs/android/lib/src/main/kotlin/breez_sdk_liquid_notification/job/InvoiceRequest.kt

```diff
@@ -59,22 +65,34 @@ class InvoiceRequestJob(
                     if (success) DEFAULT_INVOICE_REQUEST_NOTIFICATION_TITLE else DEFAULT_INVOICE_REQUEST_NOTIFICATION_FAILURE_TITLE,
                 ),
             )
+        } catch (e: PaymentException) {
+            logger.log(TAG, "Failed to process invoice request: ${e.message}", "WARN")
```

Comment:
IIUC we handle the bolt12 invoice request both on the plugin and in the sdk core.
Does this mean when the app is opened it is likely we create two invoices? (two mrh addresses, two reverse swaps to subscribe, etc..).

Reply:
If the app is already open (there is a WS connection and we are subscribed to the offer invoice requests) then Boltz only sends the invoice request to the WS (no webhook request). If there is no WS subscription, then it is only sent via webhook. If there are two instances active and both have subscribed to the offer invoice requests, Boltz only keeps the latest connection (last subscription gets the invoice request).